sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/SymmetricHashJoinStateManager.scala

```diff
@@ -476,6 +586,8 @@ class SymmetricHashJoinStateManager(
 }
 
 object SymmetricHashJoinStateManager {
+  val supportedVersions = Seq(1, 2)
```

Comment:
It would be good to just reference the same values as the config entry.

In fact, since the config is internal, you could probably just drop the check there, and rely on the checks in this code to make sure the version if correct.

Reply:
Same here. Sounds nice, but may want to keep consistency. So I'm in favor of changing all, or leaving it as it is.

Reply:
Same comment as before. I'd like the better approach to be used. If you want to go later and fix the other spots in the code, that's up to you.